dev/com.ibm.ws.jaxrs.2.0.common/src/org/apache/cxf/jaxrs/utils/JAXRSUtils.java

```diff
@@ -141,6 +142,7 @@
 import org.apache.cxf.message.MessageUtils;
 import org.apache.cxf.phase.PhaseInterceptorChain;
 import org.apache.cxf.service.Service;
+import org.codehaus.jackson.JsonParseException;
```

Comment:
I'm not sure that we want to put Jackson dependencies in the JAXRSUtils class.  Is it possible to update the Jackson MBR/MBW to throw BadRequestException instead and then leave JAXRSUtils unchanged?

Reply:
We could probably modify our wrapper class, I'd have to look into it further

Reply:
I was right, we pull in one of Jackson's prebuilt wrapper classes directly, we'd have to make our wrapper for their wrapper